package.json

```diff
@@ -3141,11 +3142,11 @@
         "@docker/sdk": "^1.0.3",
         "@grpc/grpc-js": "^1.5.0",
         "@microsoft/compose-language-service": "^0.1.1",
-        "@microsoft/vscode-azext-azureappservice": "^0.1.9",
-        "@microsoft/vscode-azext-azureutils": "^0.1.3",
-        "@microsoft/vscode-azext-utils": "^0.1.1",
+        "@microsoft/vscode-azext-azureappservice": "^0.6.4",
+        "@microsoft/vscode-azext-azureutils": "^0.3.1",
+        "@microsoft/vscode-azext-utils": "^0.3.7",
```

Comment:
There isn't really a more delicate way than to bump this package up to 0.3.7, as lesser versions have an incompatible `@vscode/extension-telemetry` package.

Reply:
Yeah. I did the same for the other ext